src/components/molecules/Form/FieldGroup/FieldGroup.tsx

```diff
@@ -15,12 +20,29 @@ const getElement = (Wrapper: React.ReactType, component?: React.ReactNode, props
   return <Wrapper {...props}>{component}&#8203;</Wrapper>;
 };
 
+const getCounterValue = (value: TValue, maxLength: TMaxLength) => {
+  let _value = 0;
+
+  if (value && typeof value === 'string') {
+    _value = value.toString().length;
+  }
+
+  return `${_value}/${maxLength}`;
+};
+
 export const FieldGroup = forwardRef<HTMLDivElement, FieldGroupProps>(
-  ({ label, error, children, ...props }, ref) => (
+  ({ label, error, children, value, maxLength, ...props }, ref) => (
     <Box ref={ref} {...props}>
-      {getElement(Label, label, { marginBottom: 1 })}
-      {React.cloneElement(children as JSX.Element, { error: !!error })}
-      {getElement(Message, error, { color: 'pinkRed', marginTop: 1 })}
+      {getElement(Label, label, { mb: 1 })}
+      {children}
+      <Flex justifyContent="space-between">
+        {getElement(Message, error, { color: 'pinkRed' })}
```

Comment:
Message teraz nie ma `margin-top`?

Reply:
hmm w sumie to chyba powinienem wynieść zarówno margin top i bottom z Message tutaj i zrobić marginY: 1 prawda? tak jak dla Label wyżej